src/lib/y2storage/setup_checker.rb

```diff
@@ -93,6 +95,32 @@ def product_warnings
       @product_warnings ||= missing_product_volumes.map { |v| SetupError.new(missing_volume: v) }
     end
 
+    def mount_warnings
+      mount_warnings = []
+
+      devicegraph.mount_points.map { |mp| mount_warning(mp) }.compact
+    end
+
+    def mount_warning(mount_point)
+      missing = mount_point.missing_mount_options
+      return if missing.empty?
+
+      msg = n_(
+        format(
+          #"Adding the fstab option %{opt} may be needed for %{path}.",
+          "The fstab option %{opt} seems to be missing for %{path}.",
+          #"The option %{opt} may be needed in fstab for %{path} to be mounted properly.",
```

Comment:
I would say: "the fstab option %{opt} may be needed for %{path} to be mounted properly". Basically option 3, but I would try to mention "fstab option" because the button for adding it is called so in the Expert Partitioner.